lib/full-site-editing/block-templates.php

```diff
@@ -208,6 +276,12 @@ function _gutenberg_build_template_result_from_post( $post ) {
 	$template->title       = $post->post_title;
 	$template->status      = $post->post_status;
 
+	if ( 'wp_template_part' === $post->post_type ) {
+		$type_terms        = get_the_terms( $post, 'wp_template_section' );
+		$section           = $type_terms[0]->name;
```

Comment:
> Also get_the_terms can return false or WP_error and we are not handling those cases here?

Oh, good point!  I wonder if this also needs to be updated for this pre-existing bit as well:

https://github.com/WordPress/gutenberg/blob/537412fd439c0c8a9befcf99e8bbbc2fdc8c1147/lib/full-site-editing/block-templates.php#L345

Reply:
How do you think we should handle the error case here?  Returning the error instead of the template probably doesn't make sense since `section` is not required.. we probably just want to not add the section field and continue to return the template?  or?

Reply:
> we probably just want to not add the section field

I'm leaning towards this.

Reply:
Cool, I updated ours to work that way.  Im still a bit unsure about what to do about that pre-existing `get_the_terms` I noted above that is used in `gutenberg_filter_wp_template_unique_post_slug`.